### PR TITLE
type: enum Effect added for severity

### PR DESCRIPTION
**Pull request by kadhikari**

### CODE REVIEW

File: type.proto [hunk 1]
```diff
@@ -137,9 +137,20 @@ message Message {
...
 message Severity {
+    enum Effect {
+        NO_SERVICE = 1;
+        REDUCED_SERVICE = 2;
+        SIGNIFICANT_DELAYS = 3;
+        DETOUR = 4;
+        ADDITIONAL_SERVICE = 5;
+        MODIFIED_SERVICE = 6;
+        OTHER_EFFECT = 7;
+        STOP_MOVED = 8;
```
Comment (pbougue) [inline type.proto]: please keep the same order (from chaos-proto)

File: type.proto [hunk 2]
```diff
@@ -137,9 +137,20 @@ message Message {
 }
 
 message Severity {
+    enum Effect {
```
Comment (pbougue) [inline type.proto]: please mention (comment) that it's a copy from chaos-proto

File: type.proto [hunk 3]
```diff
@@ -137,9 +137,20 @@ message Message {
...
+        NO_SERVICE = 1;
+        REDUCED_SERVICE = 2;
+        SIGNIFICANT_DELAYS = 3;
```
Comment (pbougue) [inline type.proto]: please add the comment on insignificant_delays

**Comment by kadhikari** [comment]
Replaced by https://github.com/CanalTP/navitia-proto/pull/112

**Pull request closed without merging**